### PR TITLE
image cache fix

### PR DESCRIPTION
**Pull request by kistitan**
Message lost fix and image cache optimazitaion

### CODE REVIEW

File: FetLife/fetlife/src/main/java/com/bitlove/fetlife/model/resource/ImageLoader.java [hunk 1]
```diff
@@ -52,7 +53,7 @@ public Response load(Uri uri, int networkPolicy) throws IOException {
     private FetLifeImageDownloader imageDowloader;
 
     public ImageLoader(Context context) {
-        imageDowloader = new FetLifeImageDownloader(context);
+        imageDowloader = new FetLifeImageDownloader(context, Integer.MAX_VALUE);
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/model/resource/ImageLoader.java]: using `9999999999` would be funnier 😋

**Comment by reneklacan** [comment]
👍

**Comment by philippt** [comment]
:+1:

**Comment by jacortinas** [comment]
:+1:

**Pull request merged**